samples/snippets/src/test/java/com/example/logging/LoggingIT.java

```diff
@@ -135,4 +134,36 @@ public void testListLogNames_shouldPass() throws Exception {
 
     assertThat(bout.toString().contains(GOOGLEAPIS_AUDIT_LOGNAME)).isTrue();
   }
+
+  @Test(timeout = 60000)
+  public void testTailLogEntriesSample() throws Exception {
+    Runnable task =
+        () -> {
+          // wait 10 seconds to allow establishing tail stream in the sample
+          try {
+            Thread.sleep(10_000);
+            try (Logging logging = LoggingOptions.getDefaultInstance().getService()) {
+              // create an instance of LogEntry with HTTP request
+              LogEntry logEntry =
+                  LogEntry.newBuilder(StringPayload.of(STRING_PAYLOAD))
+                      .setLogName(TEST_LOG)
+                      .setResource(MonitoredResource.newBuilder("global").build())
+                      .build();
+              // Writes the log entry asynchronously
+              logging.write(Collections.singleton(logEntry));
+            }
+          } catch (Exception t) {
+          }
```

Comment:
nit: something weird here with indentation - worth re-aligning so it's clearer which '{' belongs to which '}'